### PR TITLE
[ja] Translate translate concepts/storage/storage-limits into Japanese

### LINKED ISSUE #35624: [ja] Translate concepts/storage/storage-limits into Japanese
This is a Feature Request

What would you like to be added

Translate https://kubernetes.io/docs/concepts/storage/storage-limits/ into Japanese.

Why is this needed

This page has not been translated.

Comments

/language ja
/assign

### PR DESCRIPTION
**Pull request by KobayashiD27**
close #35624

### CODE REVIEW

File: content/ja/docs/concepts/storage/storage-limits.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+- Azure Disk
+- CSI
+
+ツリー内のボリュームプラグインによって管理されるボリュームの場合、Kubernetesはノードタイプを自動的に決定し、ノードに適切なボリュームの最大数を適用します。例えば：
+
+* <a href="https://cloud.google.com/compute/">Google Compute Engine</a>上では[ノードタイプ](https://cloud.google.com/compute/docs/disks/#pdnumberlimits)に応じて、最大127個のボリュームをノードに接続できます。
+
+* M5、C5、R5、T3、およびZ1DインスタンスタイプのAmazon EBSディスクの場合、Kubernetesは25ボリュームのみをノードにアタッチできます。<a href="https://aws.amazon.com/ec2/">Amazon Elastic Compute Cloud (EC2)</a>の他のインスタンスタイプの場合、Kubernetesでは39個のボリュームをノードに接続できます。
+
+* Azureでは、ノードの種類に応じて、最大64個のディスクをノードに接続できます。詳細については、[Azure の仮想マシンのサイズ](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes)を参照してください。
```
Comment (riita10069) [inline content/ja/docs/concepts/storage/storage-limits.md:53]:
```suggestion
* Azureでは、ノードの種類に応じて、最大64個のディスクをノードに接続できます。詳細については、[Azureの仮想マシンのサイズ](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes)を参照してください。
```

File: content/ja/docs/concepts/storage/storage-limits.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+<table>
+  <tr><th>クラウドサービス</th><th>ノード当たりの最大ボリューム</th></tr>
+  <tr><td><a href="https://aws.amazon.com/ebs/">Amazon Elastic Block Store (EBS)</a></td><td>39</td></tr>
+  <tr><td><a href="https://cloud.google.com/persistent-disk/">Google Persistent Disk</a></td><td>16</td></tr>
+  <tr><td><a href="https://azure.microsoft.com/en-us/services/storage/main-disks/">Microsoft Azure Disk Storage</a></td><td>16</td></tr>
+</table>
+
+## カスタム制限
+
+これらの制限を変更するには、`KUBE_MAX_PD_VOLS`環境変数の値を設定し、スケジューラを開始します。CSIドライバーの手順は異なる場合があります。制限をカスタマイズする方法については、CSIドライバーのドキュメントを参照してください。
```
Comment (t-inu) [inline content/ja/docs/concepts/storage/storage-limits.md:30]:
```suggestion
これらの制限を変更するには、`KUBE_MAX_PD_VOLS`環境変数の値を設定し、スケジューラーを開始します。CSIドライバーの手順は異なる場合があります。制限をカスタマイズする方法については、CSIドライバーのドキュメントを参照してください。
```

File: content/ja/docs/concepts/storage/storage-limits.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+{{< feature-state state="stable" for_k8s_version="v1.17" >}}
+
+動的ボリューム制限は、次のボリュームタイプでサポートされています。
+
+- Amazon EBS
+- Google Persistent Disk
+- Azure Disk
+- CSI
+
+ツリー内のボリュームプラグインによって管理されるボリュームの場合、Kubernetesはノードタイプを自動的に決定し、ノードに適切なボリュームの最大数を適用します。例えば：
```
Comment (t-inu) [inline content/ja/docs/concepts/storage/storage-limits.md:47]:
```suggestion
ツリー内のボリュームプラグインによって管理されるボリュームの場合、Kubernetesはノードタイプを自動的に決定し、ノードに適切なボリュームの最大数を適用します。例えば:
```

File: content/ja/docs/concepts/storage/storage-limits.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+* <a href="https://cloud.google.com/compute/">Google Compute Engine</a>上では[ノードタイプ](https://cloud.google.com/compute/docs/disks/#pdnumberlimits)に応じて、最大127個のボリュームをノードに接続できます。
+
+* M5、C5、R5、T3、およびZ1DインスタンスタイプのAmazon EBSディスクの場合、Kubernetesは25ボリュームのみをノードにアタッチできます。<a href="https://aws.amazon.com/ec2/">Amazon Elastic Compute Cloud (EC2)</a>の他のインスタンスタイプの場合、Kubernetesでは39個のボリュームをノードに接続できます。
+
+* Azureでは、ノードの種類に応じて、最大64個のディスクをノードに接続できます。詳細については、[Azureの仮想マシンのサイズ](https://docs.microsoft.com/en-us/azure/virtual-machines/windows/sizes)を参照してください。
+
+* CSIストレージドライバが(`NodeGetInfo`を使用して)ノードの最大ボリューム数をアドバタイズする場合、{{< glossary_tooltip text="kube-scheduler" term_id="kube-scheduler" >}}はその制限を尊重します。詳細については、[CSIの仕様](https://github.com/ontainer-storage-interface/spec/blob/master/spec.md#nodegetinfo)を参照してください。
```
Comment (t-inu) [inline content/ja/docs/concepts/storage/storage-limits.md:55]:
```suggestion
* CSIストレージドライバーが(`NodeGetInfo`を使用して)ノードの最大ボリューム数をアドバタイズする場合、{{< glossary_tooltip text="kube-scheduler" term_id="kube-scheduler" >}}はその制限を尊重します。詳細については、[CSIの仕様](https://github.com/ontainer-storage-interface/spec/blob/master/spec.md#nodegetinfo)を参照してください。
```

**Review by t-inu — COMMENTED** [review]
I made a few suggestions for notation.

**Pull request merged**